README.md

```diff
@@ -881,6 +881,10 @@ _**This is a temporary state and once a stable is released it is changed back to
 <PackageReference Include="NServiceBus.RabbitMQ" Version="4.*" />
 ```
 
+#### Version ranges
+
+Avoid using [version ranges](https://learn.microsoft.com/en-us/nuget/concepts/package-versioning#version-ranges) for package references. The integrity tests will enforce this.
```

Comment:
How is a range different from a wildcard? Then this should be avoid ranges and wildcards.

Reply:
So, we are only allowed do so a wildcard on the patch? So `1.1.*` but not `1.*` and not allowed to do `1-*` or `1.0.0-*`. 

That makes me wonder if `1.0.0-rc.*` is allowed. Assuming package restore using the highest version.

Reply:
@ramonsmits the wording is a bit fluffy because it depends, but this section clarifies that https://github.com/Particular/docs.particular.net#nuget-package-references